### ISSUE TITLE
Github URLS only Grab Zips

### ISSUE THREAD

**Issue by BaddMann** [homebysix/recipe-robot#100]
I have a few Github projects I'd like to provide, but your robot only grabs the open source zip files and not the dmg files.

Eventually I should be confident enough not to need your script to grab the correct version, but I think you guys can offer a choice via a parameter or something right? 

These projects in particular cause me issues:
https://github.com/mltframework/shotcut/releases/
https://github.com/jp9000/obs-studio

Hope this helps.

**Comment by homebysix** [comment]
Hi @BaddMann - thanks for reporting this! Parsing GitHub releases that have multiple files attached is hard, because Recipe Robot can't safely assume that "dmg" is the correct file format to look for (although the dmg is the correct file in both the examples you give).

I think one potential workaround would be to do what I suggested in [this previous issue](https://github.com/homebysix/recipe-robot/issues/98#issuecomment-216584831). (Download the correct file and feed that to Recipe Robot, and then add an `asset_regex` argument to the `GitHubReleasesInfoProvider`.)

However, I've got bad news. In both examples you give, the above workaround might solve one problem, but will reveal another:

- obs-studio is a pkg inside a dmg, which Recipe Robot often has trouble digesting.
- shotcut has neither a `CFBundleShortVersionString` nor a `CFBundleVersion` key in its Info.plist.

Bottom line: Recipes for both of the above apps will need to be written the old fashioned way, although you may be able to leverage an existing recipe as a template if you can find one for a similarly-packaged and similarly-distributed app.
